packages/mui-joy/src/CssBaseline/CssBaseline.tsx

```diff
@@ -1,10 +1,10 @@
 'use client';
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { GlobalStyles } from '@mui/system';
+import GlobalStyles from '../GlobalStyles';
```

Comment:
The [GlobalStyles from Joy](https://github.com/mui/material-ui/blob/master/packages/mui-joy/src/GlobalStyles/GlobalStyles.tsx#L9) has `themeId`, whereas `@mui/system` is not.

Reply:
Aah nice catch!